### PR TITLE
Fix for distribution of event deletions

### PR DESCRIPTION
**Pull request by annando**
Fixes https://github.com/friendica/friendica/issues/7303#issuecomment-504707887

### CODE REVIEW

File: src/Model/Item.php [hunk 1]
```diff
@@ -1080,9 +1080,11 @@ public static function deleteById($item_id, $priority = PRIORITY_HIGH)
 		}
 		// When the permission set will be used in photo and events as well,
 		// this query here needs to be extended.
-		if (!empty($item['psid']) && !self::exists(['psid' => $item['psid'], 'deleted' => false])) {
-			DBA::delete('permissionset', ['id' => $item['psid']], ['cascade' => false]);
-		}
+		// Currently deactivated. We need the permission set in the deletion process.
```
Comment (nupplaphil) [inline src/Model/Item.php]:
```suggestion
		//@todo Currently deactivated. We need the permission set in the deletion process.
```

**Comment by annando** [comment]
I currently messed up my repository a little bit. I need to clean this up, but need some breakfast as well. So this will take some time ;-)

**Comment by MrPetovan** [comment]
Do you need more time for this PR? The release is waiting.

**Comment by annando** [comment]
I think this PR is done.

**Comment by AlfredSK** [comment]
Works! Fixes also #7241

**Pull request merged**